pkg/apis/logging/v1alpha1/types.go

```diff
@@ -40,14 +40,14 @@ type VisualizationSpec struct {
 }
 
 type KibanaSpec struct {
-	Resources    v1.ResourceRequirements `json:"resources"`
-	NodeSelector map[string]string       `json:"nodeSelector,omitempty"`
-	Replicas     int32                   `json:"replicas"`
+	Resources    *v1.ResourceRequirements `json:"resources"`
```

Comment:
is there a reason we're moving to use a pointer here? We don't do this in the EO currently

Reply:
Move to a pointer to know if someone specifies resource or not (via nil).  This is a clean way to know if we use what is provided or sane default